terraform/ipfs-mirror/nginx-default

```diff
@@ -42,9 +42,17 @@ server {
     # First attempt to serve request as file, then
     # as directory, then fall back to displaying a 404.
     try_files $uri $uri/ =404;
+    
+    # Prevent Caching
+    expires           0;
```

Comment:
There are some discussions about `0` as the value, vs. valid date in the past. A little concerned about browser compat.
https://stackoverflow.com/questions/11357430/http-expires-header-values-0-and-1

Reply:
We will keep this as `0`